aws-codeartifact-repository/src/main/java/software/amazon/codeartifact/repository/DeleteHandler.java

```diff
@@ -23,21 +24,30 @@
 
 
         this.logger = logger;
+        ResourceModel model = request.getDesiredResourceState();
 
         return ProgressEvent.progress(request.getDesiredResourceState(), callbackContext)
             .then(progress ->
                 proxy.initiate("AWS-CodeArtifact-Repository::Delete", proxyClient, progress.getResourceModel(), progress.getCallbackContext())
                     // STEP 2.1 [construct a body of a request]
                     .translateToServiceRequest(Translator::translateToDeleteRequest)
                     // STEP 2.2 [make an api call]
-                    .makeServiceCall((awsRequest, client) -> deleteRepository(progress, client, awsRequest))
+                    .makeServiceCall((awsRequest, client) -> {
+                        // if domain does not exist, deleteDomain does not throw an exception, so we must do this
```

Comment:
I think this comment should be about the repository, not domain?

Reply:
yes, copy pasta :(